lldb/include/lldb/Target/Thread.h

```diff
@@ -1336,8 +1338,18 @@ class Thread : public std::enable_shared_from_this<Thread>,
     return StructuredData::ObjectSP();
   }
 
+<<<<<<< HEAD
```

Comment:
Would this go away if I wasn't looking at the patches one by one?

Reply:
Yes